addons/pos_sale/models/pos_order.py

```diff
@@ -124,6 +124,29 @@ def action_view_sale_order(self):
             'domain': [('id', 'in', linked_orders.ids)],
         }
 
+    def _prepare_tax_base_line_values(self, sign=1):
+        res = super()._prepare_tax_base_line_values(sign)
+        for line in res:
+            pos_order_line = line['record']
+            origin_order = pos_order_line.sale_order_origin_id
+            if origin_order:
+                #This is based on https://github.com/odoo/odoo/blob/7e3403068fc3fbc40182b3cfeb80e97a9300e8ff/addons/sale/models/sale_order_line.py#L1081
+                analytic_account_id = origin_order.analytic_account_id.id
+                origin_line = pos_order_line.sale_order_line_id
+                if origin_line.analytic_distribution and not origin_line.display_type:
+                    line['analytic_distribution'] = origin_line.analytic_distribution
+                if analytic_account_id and not origin_line.display_type:
```

Comment:
```suggestion
                if origin_line.display_type:
                    continue

                if origin_line.analytic_distribution:
                    line['analytic_distribution'] = origin_line.analytic_distribution

                if analytic_account_id:
```

May I know why does display_type matter in this calculation?